frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableGroup/ticketsTableGroup.component.tsx

```diff
@@ -18,36 +18,45 @@
 import { ITicket } from '@/v5/store/tickets/tickets.types';
 import { FormattedMessage } from 'react-intl';
 import AddCircleIcon from '@assets/icons/filled/add_circle-filled.svg';
-import { Transformers, useSearchParam } from '@/v5/ui/routes/useSearchParam';
 import { isCommenterRole } from '@/v5/store/store.helpers';
-import { ProjectsHooksSelectors } from '@/v5/services/selectorsHooks';
 import { useContext } from 'react';
 import { SortedTableComponent, SortedTableContext, SortedTableType } from '@controls/sortedTableContext/sortedTableContext';
 import { BaseProperties, IssueProperties, SafetibaseProperties } from '@/v5/ui/routes/viewer/tickets/tickets.constants';
 import ArrowIcon from '@assets/icons/outlined/arrow-outlined.svg';
-import { Header, Headers, Group, NewTicketRow, NewTicketText, IconContainer } from './ticketsTableGroup.styles';
+import { Table, Header, Headers, Group, NewTicketRow, NewTicketText, IconContainer, PlaceholderForStickyFunctionality } from './ticketsTableGroup.styles';
 import { TicketsTableRow } from './ticketsTableRow/ticketsTableRow.component';
 import { NewTicketMenu } from '../../newTicketMenu/newTicketMenu.component';
 import { useSelectedModels } from '../../newTicketMenu/useSelectedModels';
 import { SetTicketValue } from '../../ticketsTable.helper';
-import { useParams } from 'react-router-dom';
-import { DashboardTicketsParams } from '@/v5/ui/routes/routes.constants';
+import { ResizableTableCell } from '@controls/resizableTableContext/resizableTableCell/resizableTableCell.component';
+import { ResizableTableContext } from '@controls/resizableTableContext/resizableTableContext';
 
-const SortingTableHeader = ({ name = null, children, hidden = false, ...props }) => {
+const SortingTableHeader = ({ name, children, disableSorting = false, ...props }) => {
```

Comment:
Im not sure why this originally was not a change in the base component `<Header />` . If we can I would move the original changes to Header if they don't clash with other use of the component and create a ResizableHeader in yhr trsizable table direactory to be resuse.

Reply:
A `ResizableTableHeader` would just be the same as `ResizableTableCell`, so there wouldn't be any real benefits.

What could be done instead, is to extract the `SortingTableHeader` as that would make sense to exist an independent component which part of the `SortingTableContext` ecosystem. This way we can then refactor other dashboard tables to use it as well. However, I would leave this to another issue